compiler/luci/service/src/Nodes/CircleStridedSlice.cpp

```diff
@@ -403,16 +398,33 @@ loco::TensorShape Algorithm::visit(const luci::CircleStridedSlice *node)
   LUCI_ASSERT(end_node->rank() == 1, "Only support rank 1 for end_node");
   LUCI_ASSERT(strides_node->rank() == 1, "Only support rank 1 for strides_node");
 
+  auto begin_const = dynamic_cast<luci::CircleConst *>(node->begin());
+  auto end_const = dynamic_cast<luci::CircleConst *>(node->end());
+  auto strides_const = dynamic_cast<luci::CircleConst *>(node->strides());
+  // TODO support non-const strides_node
+  if (strides_const == nullptr)
+  {
+    INTERNAL_EXN("StridedSlice strides node are not Constant");
+  }
+  // TODO Support cases where the mask attributes are non-zero.
```

Comment:
Do we need to consider this attributes when `begin/end` is not const?

I'm asking this question because I don't think there's a need to add this comment here,
if the attribute is not related to the if statement below.

Reply:
I thought that when the mask was set, some parts of the shape could be fixed. However, since there are no issues with inference now, it would be better to remove the comment.